File: vestibulapp_service/VestibulApp.Core/Inscricao.php
<?php
    include_once '../../VestibulApp.Data/Connect.php';
    include_once '../../VestibulApp.Data/Insert.php';
    include_once '../../VestibulApp.Data/Delete.php';
    include_once '../../VestibulApp.Data/Update.php';
    include_once '../../VestibulApp.Data/SelectAll.php';
    include_once '../../VestibulApp.Data/SelectById.php';
    include_once '../../VestibulApp.Data/SelectBySpecification.php';
    
    class Inscricao{
        private $id_curso;
        private $id_vestibular;
        private $id_candidato;
        private $lingua; 

        private $connection;

        public function openConnect(){
            $this->connection = new Connect();
            $this->connection->connect();
        }

        public function disconnect(){
            $this->connection->closeConnection();
        }

       	public function getId_curso(){
            return $this->id_curso;
        }

        public function setId_curso($id_curso){
            $this->id_curso = $id_curso;
        }

        public function getId_vestibular(){
            return $this->id_vestibular;
        }

        public function setId_vestibular($id_vestibular){
            $this->id_vestibular = $id_vestibular;
        }

        public function getId_candidato(){
            return $this->id_candidato;
        }

        public function setId_candidato($id_candidato){
            $this->id_candidato = $id_candidato;
        }

        public function getLingua(){
            return $this->lingua;
        }

        public function setLingua($lingua){
            $this->lingua = $lingua;
        }

        public function Insert(){
            try{               
                $insert = new Insert();
                $insert = $insert->Inscricao($this->id_curso, $this->id_vestibular, $this->id_candidato, $this->lingua);

                return mysqli_query($this->connection->getMySqli(), $insert);
            }catch(Exception $ex){
                return $ex;
            }
        } 

        public function SelectBySpecification(){
            try{
                $selectBySpecification = new SelectBySpecification();
                $selectBySpecification = $selectBySpecification->Inscricao(" AND id_vestibular = " .$this->id_vestibular);

                $resultado = mysqli_query($this->connection->getMySqli(), $selectBySpecification);

                if(mysqli_num_rows($resultado) > 0){
                    while($row =  mysqli_fetch_assoc($resultado)) {
                        $select[] = $row['id_inscricao'];
                        $select[] = $row['id_curso'];
                        $select[] = $row['id_vestibular'];
                        $select[] = $row['id_candidato'];
                        $select[] = $row['lingua'];
                        return $select;
                    }
                }
                else{
                    return false;
                }
            }catch(Exception $ex){
                return $ex;
            }
        }
    }
?>

File: vestibulapp_app/app/src/main/java/com/example/alexalves/vestibulapp/Entidades/Inscricao.java
package com.example.alexalves.vestibulapp.Entidades;

/**
 * Created by Desenvolvedor on 04/11/2016.
 */

public class Inscricao {

    private int idInscricao;
    private int idCurso;
    private int idVestibular;
    private int idCandidato;
    private String lingua;

    public Inscricao(){

    }

    private static Inscricao holder;

    public int getIdInscricao() {
        return idInscricao;
    }

    public void setIdInscricao(int idInscricao) {
        this.idInscricao = idInscricao;
    }

    public int getIdCurso() {
        return idCurso;
    }

    public void setIdCurso(int idCurso) {
        this.idCurso = idCurso;
    }

    public int getIdVestibular() {
        return idVestibular;
    }

    public void setIdVestibular(int idVestibular) {
        this.idVestibular = idVestibular;
    }

    public int getIdCandidato() {
        return idCandidato;
    }

    public void setIdCandidato(int idCandidato) {
        this.idCandidato = idCandidato;
    }

    public String getLingua() {
        return lingua;
    }

    public void setLingua(String lingua) {
        this.lingua = lingua;
    }

    public static Inscricao getInscricao() {
        return holder;
    }

    public static void setInscricao(Inscricao holder) {
        Inscricao.holder = holder;
    }
}
